lib/from_openstudio/material/opaque_no_mass.rb

```diff
@@ -0,0 +1,54 @@
+# *******************************************************************************
+# Honeybee OpenStudio Gem, Copyright (c) 2020, Alliance for Sustainable
+# Energy, LLC, Ladybug Tools LLC and other contributors. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+# (1) Redistributions of source code must retain the above copyright notice,
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'honeybee/material/opaque_no_mass.rb'
+require 'to_openstudio/model_object'
+
+module Honeybee
+  class EnergyMaterialNoMass < ModelObject
+
+    def self.from_material(material)
+        # create an empty hash
+        hash = {}
+        hash[:type] = 'EnergyMaterialNoMass'
+        # set hash values from OpenStudio Object
+        hash[:display_name] = material.nameString
+        hash[:r_value] = material.thermalResistance
+        hash[:roughness] = material.roughness
+        hash[:thermal_absorptance] = material.thermalAbsorptance
+        hash[:solar_absorptance] = material.solarAbsorptance
+        hash[:visible_absorptance] = material.visibleAbsorptance
```

Comment:
If I am not mistaken these three variables are `boost::optionals` no?

![image](https://user-images.githubusercontent.com/5567574/113190113-f532e580-9229-11eb-83f6-ba3fc84751df.png)

So don't you need to do something like
```
unless material.thermalAbsorptance.empty?
  hash[:thermal_absorptance] = material.thermalAbsorptance.get
end
```

Reply:
@chriswmackey Sorry I have added checks for Boost::Optional properties in some translators I haven't committed yet but forgot to go through these existing translators. I can add this.
While creating the forward translators from we checked for user defined properties or added default values from the schema. Some of these optional properties have defaults in the schema, I am not sure we want to add from those for the reverse translators as well?